buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -225,35 +656,44 @@
       "value": null,
       "isSecret": true
     },
-    "PB_VsoRepoUrl": {
-      "value": "--branch $(PB_Branch) https://$(PB_VsoAccountName):$(PB_VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Core-Setup-Trusted"
-    },
     "PB_VsoAccountName": {
       "value": "dn-bot"
     },
     "PB_VsoPassword": {
       "value": null,
       "isSecret": true
     },
+    "PB_Branch": {
```

Comment:
Should use $(SourceBranch) instead.  See [https://github.com/dotnet/core-setup/pull/2460/files](https://github.com/dotnet/core-setup/pull/2460/files) for how it doesn't work.

---
In reply to: [118108233](https://github.com/dotnet/core-setup/pull/2477#discussion_r118108233) [](ancestors = 118108233)